benchmark/utils/nodeRedUtils.ts

```diff
@@ -0,0 +1,144 @@
+import http from 'http';
+import * as dotenv from 'dotenv';
+dotenv.config();
```

Comment:
This can be simplified as follows:

```suggestion
import 'dotenv/config'
```

Reply:
Done